flink-connectors/flink-connector-base/src/test/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriterTest.java

```diff
@@ -63,13 +64,7 @@ public void before() {
     private void performNormalWriteOfEightyRecordsToMock()
             throws IOException, InterruptedException {
         AsyncSinkWriterImpl sink =
-                new AsyncSinkWriterImplBuilder()
```

Comment:
These changes are applied in a separate commit, if they make the diff hard to read please look at commit